website/components/PluginsTable.tsx

```diff
@@ -0,0 +1,77 @@
+import { Table, Tr, Td, Th } from "nextra/components";
+import Link from "next/link";
+import { getLatestVersion } from "../utils/versions";
+
+const TableLink = ({ text, href, target = undefined }) => {
+  return (
+    <Link href={href}>
+      <a
+        className="nx-text-primary-500 nx-underline nx-decoration-from-font [text-underline-position:under]"
```

Comment:
Once https://github.com/shuding/nextra/issues/332 is fixed we should be able to not hard code the classes